src/main/kotlin/no/nav/amt/person/service/controller/PersonController.kt

```diff
@@ -0,0 +1,61 @@
+package no.nav.amt.person.service.controller
+
+import no.nav.amt.person.service.controller.dto.*
+import no.nav.amt.person.service.controller.request.ArrangorAnsattRequest
+import no.nav.amt.person.service.controller.request.NavAnsattRequest
+import no.nav.amt.person.service.controller.request.NavBrukerRequest
+import no.nav.amt.person.service.controller.request.NavEnhetRequest
+import no.nav.amt.person.service.nav_ansatt.NavAnsattService
+import no.nav.amt.person.service.nav_bruker.NavBrukerService
+import no.nav.amt.person.service.nav_enhet.NavEnhetService
+import no.nav.amt.person.service.person.PersonService
+import no.nav.amt.person.service.person.RolleService
+import no.nav.amt.person.service.person.model.Rolle
+import no.nav.security.token.support.core.api.Protected
+import org.springframework.web.bind.annotation.*
+
+
+@RestController
+@RequestMapping("/api")
+class PersonController (
+	private val personService: PersonService,
+	private val navAnsattService: NavAnsattService,
+	private val navBrukerService: NavBrukerService,
+	private val navEnhetService: NavEnhetService,
+	private val rolleService: RolleService,
+) {
+
+	@Protected
```

Comment:
Her kan du gjerne legge til hva som er forventet issuer, om det er tokenx eller azuread, f.eks. sånn: `@ProtectedWithClaims(issuer = Issuer.TOKEN_X)` :)

Reply:
Hvis disse skal kalles med en sluttbrukerkontekst, dvs, at kallene opprinnelig kommer fra arrangør-flaten, så kan du gjerne bruke tokenx her. Da kan du hente ut fnr/personident direkte fra tokenet. Fordelen med det er at man slipper å kontrollere at fnr i token stemmer overens med fnr i bodyen (hvis man ikke sjekker dette risikerer man å gjøre det mulig å hente opplysninger om andre enn seg selv så lenge man er innlogget).

Reply:
Det er det jeg ikke er helt sikker på, det avhenger vel litt av hvordan bruken blir. Stort sett nå noen ressurser her skal opprettes gjøres vel det (for nå) når det kommer en melding om en deltaker fra arena-acl til amt-tiltak og det virker logisk med en machine-to-machine token (ikke om jeg helt vet om det er det jeg har lagd her da 😅). Men det er godt mulig jeg tenker feil her, eller det er andre behov.

Når det kommer til arrangør ansatte så kan det hende det skal være tokenx, flyten er vel at noen logger seg inn første gang så sender de inn sitt fnr i tokenet og at det skulle vært sendt inn hit.

Reply:
Enig i den!